lib/_debugger.js

```diff
@@ -1641,7 +1643,7 @@ Interface.prototype.trySpawn = function(cb) {
         process._debugProcess(pid);
       } catch (e) {
         if (e.code === 'ESRCH') {
-          console.error(`Target process: ${pid} doesn't exist.`);
+          console.error(prefix + pid + `Target process: ${pid} doesn't exist.`);
```

Comment:
Careful here, `pid` is defined in this inner scope, overshadowing your top-scope definition. Won't be an issue when you use a function, though :)